apps/browser/src/fido2/popup/fido2/fido2.component.html

```diff
@@ -0,0 +1,58 @@
+<!-- TODO: Rewrite and refactor this component when implementing the new design -->
+<ng-container *ngIf="data$ | async as data">
+  <div class="auth-wrapper">
+    <i class="bwi bwi-spinner bwi-lg bwi-spin" [hidden]="!loading" aria-hidden="true"></i>
+    <ng-container *ngIf="data.showUnsupportedVerification">
+      Verification required by the initiating site. This feature is not yet implemented for accounts
+      without master password.
+    </ng-container>
+    <ng-container *ngIf="!data.showUnsupportedVerification">
+      <ng-container
+        *ngIf="
+          data.message.type == 'PickCredentialRequest' ||
+          data.message.type == 'ConfirmNewNonDiscoverableCredentialRequest'
+        "
+      >
+        A site is asking for authentication, please choose one of the following credentials to use:
+        <div class="box list">
+          <div class="box-content">
+            <app-cipher-row
+              *ngFor="let cipher of ciphers"
+              [cipher]="cipher"
+              (onSelected)="pick(cipher)"
+            ></app-cipher-row>
+          </div>
+        </div>
+      </ng-container>
+      <ng-container *ngIf="data.message.type == 'ConfirmNewCredentialRequest'">
+        A site wants to create the following passkey in your vault
+        <div class="box list">
+          <div class="box-content">
+            <app-cipher-row [cipher]="ciphers[0]"></app-cipher-row>
+          </div>
+        </div>
+        <button type="button" class="btn btn-outline-secondary" (click)="confirm()">Create</button>
+      </ng-container>
+      <ng-container *ngIf="data.message.type == 'InformExcludedCredentialRequest'">
+        A passkey already exists in Bitwarden for this account
+        <div class="box list">
+          <div class="box-content">
+            <app-cipher-row *ngFor="let cipher of ciphers" [cipher]="cipher"></app-cipher-row>
+          </div>
+        </div>
+      </ng-container>
+      <ng-container *ngIf="data.message.type == 'InformCredentialNotFoundRequest'">
+        You do not have a matching login for this site.
+      </ng-container>
+    </ng-container>
+    <button
+      *ngIf="data.fallbackSupported"
+      type="button"
+      class="btn btn-outline-secondary"
+      (click)="abort(true)"
+    >
+      Use browser built-in
+    </button>
+    <button type="button" class="btn btn-outline-secondary" (click)="abort(false)">Abort</button>
```

Comment:
```suggestion
    <button class="btn btn-outline-secondary" (click)="abort(false)">Abort</button>
```
Buttons have an inherent type of `button`. It's not necessary to set that here. You could otherwise use `type="button"` on something like a `<div>` or even an `<a>` that acts as a button rather than a link.

Reply:
The default type for buttons is actually `submit` ([reference](https://www.w3.org/TR/html401/interact/forms.html#h-17.5)), which can lead to issues if there is a button that is not a submit type somewhere within a form. We have been going through and adding `type="button"` to ones that are not submit buttons (see [PM-1399](https://bitwarden.atlassian.net/browse/PM-1399))

Reply:
@differsthecat 

Was working through my BEEEP today and saw the lint rule. I'm completely wrong on that, hah. I think I was thinking about the implicit `role` of the button, but you're absolutely right the implicit default `type` is a `submit` type. 

x_x

Reply:
No worries at all @cagonzalezcs ! Defaulting to `button` would be my assumption too if I hadn't seen the ticket but unfortunately, that is not the case 🫠